src/components/30-organisms/footer/index.js

```diff
@@ -51,6 +58,78 @@ class AXAFooter extends LitElement {
     this._accordionActiveIndex = -1;
   }
 
+  firstUpdated() {
+    const resetStyle = document.createElement('style');
+    resetStyle.textContent = AXAFooter.resetHeadingCss;
+    this.appendChild(resetStyle);
+  }
+
+  prepareSlotsWithIndexes() {
+    // set slotted numbers automatically for all titles and all items for each column
+    // every title followed with items will be a column (dome sequence)
+    // There is no wrapper for the columns but a counter that controls which
+    // column to use
+    // EXAMPLE IN PSEUDO DOM:
+    //
+    // TITLE -> start column 0
+    // ITEM -> add item to column 0
+    // ITEM -> add item to column 0
+    // ITEM -> add item to column 0
+    // ITEM -> add item to column 0
+    //
+    // TITLE -> start column 1
+    // ITEM -> add item to column 1
+    // ITEM -> add item to column 1
+    // ITEM -> add item to column 1
+
+    const childrenArr = Array.prototype.slice.call(this.children);
+
+    const filter = criteria => child =>
+      ~child.getAttribute('slot').indexOf(criteria);
+
+    const noHeaderFilter = criteria => child => {
+      const { nodeName } = child;
+      return (
+        filter(criteria)(child) && !~HEADINGS.indexOf(nodeName.toLowerCase())
```

Comment:
you can also use `string.includes`, we have it in the polyfills

Reply:
done